_data/docs-home.yml

```diff
@@ -100,6 +100,8 @@ toc:
     path: /docs/user-guide/integrations/tti/
   - title: LORIOT
     path: /docs/user-guide/integrations/loriot/
+  - title: ChirpStack
```

Comment:
Add the same to docs-home.yml for Cloud.

Reply:
Added.